rest_api/__init__.py

```diff
@@ -105,23 +80,6 @@ async def index():
     }, 200
 
 
-@app.get("/favicon.ico")  # Favicon, my ass. We need no favicon for an API.
-@hide
-async def favicon_my_ass():
-	return "", 200
-
-
-@app.get("/status")
-async def get_status():
-    return {
-        "scratchDeprecated": True,
-        "registrationEnabled": app.supporter.registration,
-        "isRepairMode": app.supporter.repair_mode,
-        "ipBlocked": (blocked_ips.search_best(request.ip) is not None),
-        "ipRegistrationBlocked": (registration_blocked_ips.search_best(request.ip) is not None)
-    }, 200
-
-

```

Comment:
Why remove /status, there's no reason to remove it

Reply:
Yup, no reason to remove it.

Reply:
scratchDeprecated is pointless now, registrationEnabled can be added to / if you care so much, and repair mode / ip blocks give errors for the rest of the api

Reply:
> scratchDeprecated is pointless now, registrationEnabled can be added to / if you care so much, and repair mode / ip blocks give errors for the rest of the api

womp womp just keep it no need to remove it